frontend/src/scenes/groups/GroupsIntroduction.tsx

```diff
@@ -12,8 +12,11 @@ export function GroupsIntroduction(): JSX.Element {
             docsLink="https://posthog.com/docs/user-guides/group-analytics"
         >
             <div className="flex flex-col items-center mt-4 justify-center text-center border rounded-lg py-8 min-h-56">
-                <h2 className="mb-2 text-2xl font-semibold">You're almost done!</h2>
-                <div className="max-w-140">Learn how to track groups in your code</div>
+                <h2 className="mb-2 text-2xl font-semibold">Start tracking groups</h2>
+                <div className="max-w-140">
+                    Get a 360&deg; view of how companies or teams use your product. Use the SDK to identify each group,
+                    and then include the relevant group identifier in the event&nbsp;properties.
```

Comment:
Is "identify each group" here referring to "creating a group" or "creating group type" ?

I feel like people won't really know what "identify" means

```suggestion
                    Get a 360&deg; view of how companies or teams use your product. Use the SDK to create a group type,
                    and then include the group identifier in the event&nbsp;properties.
```

Reply:
Fair enough. I went with "create a group" 4707e53668